pkg/helmclient/lister.go

```diff
@@ -0,0 +1,64 @@
+package helmclient
+
+import (
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+// ListResult is the result of the list operation
+type ListResult struct {
+	Releases []*Release
+}
+
+// NewListResult returns an instance of ListResult.
+// It format the result according to the contract
+func NewListResult(releases []*release.Release) *ListResult {
+	rels := []*Release{}
```

Comment:
You never need slice of pointers as that's a reference already. Infact try to avoid pointers and unless you really need it don't add.